src/style/index.css

```diff
@@ -1306,4 +1306,39 @@ input[type="number"] {
     font-weight: 700;
     color: #2856ff;
     font-size: 1rem;
-}
\ No newline at end of file
+}
+
+
+
+@media print {
+    body {
+      padding-top: 40px;
+      padding-bottom: 40px;
+    }
```

Comment:
This padding is already applied by the `PrintPreview` component isn't it?

Reply:
Yes, you're correct. The PrintPreview component already applies padding with the p-10 class. The additional padding defined in the print styles (print:p-2) would override this padding when printing.